packages/select/src/common/listItemsProps.ts

```diff
@@ -28,7 +28,7 @@ export interface IListItemsProps<T> extends IProps {
      * uncontrolled (managed by the component's state). Use `onActiveItemChange`
      * to listen for updates.
      */
-    activeItem?: T | null;
+    activeItem?: IQueryListActiveItem<T> | null;
```

Comment:
This is a breaking change, no? Also, I don't love this API. Is there a way for us to just accept `T | null`, or `T | IQueryListCreateItem<T> | null`, or something?

Reply:
I agree, this is a break and we can't merge as-is. I don't see a way to make this non-breaking if we try to adjust the existing prop types (we can make the addition to `activeItem` non-breaking but not `onActiveItemChange`)... so we'll have to add new optional props for this behavior. we could consider refactoring them to make the API more streamlined in the next major version, but for now I propose:

```ts
isCreateNewItemActive: boolean;

onCreateNewItemActiveChange: () => null;
```

users will have to reconcile for themselves what happens between `onActiveItemChange` and `onCreateNewItemActiveChange`